app/views/accounts/_auth_apps.html.erb

```diff
@@ -1,18 +1,8 @@
-<div class="grid-row margin-bottom-1 margin-top-0">
-  <h2 class="grid-col-fill margin-y-0 padding-right-2">
+<div class="margin-top-0 margin-bottom-1">
+  <h2 class="margin-y-0">
```

Comment:
A `div` would have no margin of its own so `margin-top-0` isn't necessary.

```suggestion
<div class="margin-bottom-1">
  <h2 class="margin-y-0">
```

Or we could make this consistent with the other files where we got rid of the `div` wrapper altogether.

```suggestion
<h2 class="margin-top-0 margin-bottom-1">
```